### DISCUSSION TITLE
DBA DASH DB table sync configuration settings

### DISCUSSION THREAD

**Discussion by phaninvm** [trimble-oss/dba-dash#958] [General]
Hi

we have set up an alert logic when ever any database / snapshot in suspect mode, when snapshot was recreated on my DB the same is getting reflecting after 1 hour. Can any one please help me in changing the dbadash configuration settings to sync data from dbo.databases table on DBA DASH database for every 1 minute.

**Comment by DavidWiseman** [comment]
Hi, you can adjust the schedule for the Databases collection to run every 1min instead of every hour.  Schedule customization is [documented here](https://dbadash.com/docs/help/schedule/#schedule-customization).  Once you've changed the schedule save the config and restart the service to pick up the change.

Hope this helps - reply back if you need help.
